zScanner/ViewControllers/Documents/DocumentsListViewController.swift

```diff
@@ -46,6 +46,7 @@ class DocumentsListViewController: BaseViewController, ErrorHandling {
         super.viewWillAppear(animated)
         
         documentsViewModel.updateDocuments()
+        documentsViewModel.deleteOldSentDocuments()
```

Comment:
I don't like the name. What do you think about `deleteExpiredDocuments` ? 
It's up to discussion.